test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/Multiple.Schema.OpenApi.V2.json

```diff
@@ -928,12 +928,7 @@
         ],
         "summary": "Create new navigation property ref to Revisions for Documents",
         "operationId": "Documents.CreateRefRevisions",
-        "consumes": [
```

Comment:
I'm a bit confused here, shouldn't it still consume JSON?

Reply:
It should consume JSON. The v2 deserializer is not properly handling ref requestBodies. I've created this [issue](https://github.com/microsoft/OpenAPI.NET/issues/932) to track the problem.

Reply:
thanks for logging the issue, so this is "not this lib's problem" at this point. resolving.